realm/realm-library/src/main/cpp/io_realm_internal_Util.cpp

```diff
@@ -37,6 +38,9 @@ const string TABLE_PREFIX("class_");
 
 JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM* vm, void*)
 {
+    // Workaround some known bugs of system calls on some special devices.
```

Comment:
I would rewrite this to:

"Workaround for some known bugs in system calls on specific devices"